ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -4966,4 +4975,13 @@ EFHLC:
   - low_exhaust_fan_speed_status
   - high_exhaust_fan_speed_status
   implements:
-  - OPERATIONAL
\ No newline at end of file
+  - OPERATIONAL
+
+SWCM:
```

Comment:
i would be expecting valve information on the fields, not run commands (which are used to tell pieces of equipment to operate). also, without some context for the devices themselves its hard to say what the least necessary set of subfields will be. is this a stand-alone solenoid valve (and is it really in the HVAC namespace) or is it part of another piece of equipment.